### DISCUSSION TITLE
Go to Reddit to view mature content?

### DISCUSSION THREAD

**Discussion by hashnag** [Docile-Alligator/Infinity-For-Reddit#1536] [Q&A] [answered]
Now that this is a paid app will there be the ability to read subs marked mature again?

I figured it was just a way for Reddit to annoy 3rd party client users into paying for API access but today I still see the warning on adult themed subs (as a subscriber).

Thanks!

**Comment by voxvargr** [comment, marked as answer]
Afaik that hasn't changed. Reddit closed all nsfw API even for paid access. They claimed that's for protection so reddit can verify who sees it is of age afaik
